feat: CXSPA-1249 NgBootstrap removal from close account components

pawelfras opened the pull request.

pawelfras committed:
feat: CXSPA-1249 NgBootstrap removal from close account components

pawelfras committed:
unit test for click outside modal

Commit:
Merge branch 'epic/ngbootstrap-removal' of github.com:SAP/spartacus into feature/CXSPA-1249

Commit:
Merge branch 'epic/ngbootstrap-removal' of github.com:SAP/spartacus into feature/CXSPA-1249

morganm58 commented on feature-libs/user/profile/components/close-account/components/close-account-modal/close-account-modal.component.html:
```diff
@@ -1,43 +1,50 @@
+          </p>
+        </div>
+        <div class="cx-close-account-modal-footer">
           <button class="btn btn-primary" (click)="closeAccount()">
```
this should also have the `btn-block ` class

morganm58 committed:
feature/CXSPA-1249
added cx-close-account selector to the index file and also added class change to the secondary button

morganm58 committed:
feature/CXSPA-1249
updated styling for close account modal
